src/Raven.Server/ServerWide/Commands/CommandBase.cs

```diff
@@ -9,6 +11,8 @@ namespace Raven.Server.ServerWide.Commands
 {
     public abstract class CommandBase
     {
+        internal const string MinBuildVersion54200 = "5.4.200";
```

Comment:
Why this is 5.4.200? We have released such version already

Reply:
I am using the AssemblyVersion. We don't have the full build version in development, so the checks and tests are failing.